### PR TITLE
Button

### PR DESCRIPTION
**Pull request by Rockyraven**

### CODE REVIEW

File: component/button/button.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ *{
+     margin: 0px;
+     padding: 0px;
+    
+ }
+ .container{
+     display: flex;
+     height: 8rem;
+     width: 80%;
+     /* background-color: aqua; */
```
Comment (Shubhamkr2610) [inline component/button/button.css]: remove unused code

File: component/button/button.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+     border: 1px solid;
+     height: 2rem;
+     padding: 5px;
+     border-radius: 5px;
+      
+     font-weight: bold;
+     justify-content: space-around;
+ }
+ .standard-primary {
+     background-color: rgb(99, 99, 241);
```
Comment (arjun-computer-geek) [inline component/button/button.css]: use color variable instead actual value

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]

File: component/card/card.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@import url('/style.css');
+ @import url('/component/slidebaar.css');
+ @import url('/component/alert/alert.css');
+
```
Comment (Shubhamkr2610) [inline component/card/card.css:4]: I think alert.css shouldn't be in card.css

File: component/button/button.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+     
+ }
+ .standard-secondary {
+     background-color: rgb(154, 243, 154);
+ }
+ .standard-error {
+     background-color: rgb(250, 106, 106);
+ }
+ .standard-warning {
+     background-color: rgb(248, 248, 66);
```
Comment (Shubhamkr2610) [inline component/button/button.css]: use color variable name

**Review by arjun-computer-geek — APPROVED** [review]
looks good

File: component/button/button.css [hunk 5]
```diff
@@ -39,7 +40,7 @@
      background-color: rgb(250, 106, 106);
  }
  .standard-warning {
-     background-color: rgb(248, 248, 66);
+     background-color: rgb(170, 170, 0);
  }
  .outline-button{
      display: flex;
```
Comment (Shubhamkr2610) [inline component/button/button.css:46]: remove extra spacing

**Review by Shubhamkr2610 — APPROVED** [review]
looks good  to me.

**Pull request merged**